lib/improver/tests/lapse_rate/Test_LapseRate.py

```diff
@@ -290,6 +344,85 @@ def test_lapse_rate_limits(self):
+        """Test that the function correctly applies a specified, non default
+        maximum lapse rate."""
+        reset_cube_data(self.temperature, self.orography, self.land_sea_mask)
+
+        expected_out = np.array([[0.0392, 0.0392, 0.0, DALR, DALR],
+                                 [0.0392, 0.0392, 0.0, DALR, DALR],
+                                 [0.0392, 0.0392, 0.0, DALR, DALR],
+                                 [0.0392, 0.0392, 0.0, DALR, DALR],
+                                 [0.0392, 0.0392, 0.0, DALR, DALR]])
+
+        # West data points should be -4*DALR and East should be DALR.
+        self.temperature.data[:, :, 0] = 2
+        self.temperature.data[:, :, 1] = 1
+        self.temperature.data[:, :, 3] = -1
+        self.temperature.data[:, :, 4] = -2
+        self.orography.data[:, :] = 10
+
+        result = LapseRate(nbhood_radius=1,
+                           max_lapse_rate=-4 * DALR).process(
+            self.temperature,
+            self.orography,
+            self.land_sea_mask)
+
+        self.assertArrayAlmostEqual(result.data, expected_out)
+
+    def test_specified_min_lapse_rate(self):
+        """Test that the function correctly applies a specified, non default
+        minimum lapse rate."""
+        reset_cube_data(self.temperature, self.orography, self.land_sea_mask)
+
+        expected_out = np.array([[0.0294, 0.0294, 0.0, -0.0196, -0.0196],
+                                 [0.0294, 0.0294, 0.0, -0.0196, -0.0196],
+                                 [0.0294, 0.0294, 0.0, -0.0196, -0.0196],
+                                 [0.0294, 0.0294, 0.0, -0.0196, -0.0196],
+                                 [0.0294, 0.0294, 0.0, -0.0196, -0.0196]])
+
+        # West data points should be -3*DALR and East should be 2*DALR.
+        self.temperature.data[:, :, 0] = 2
+        self.temperature.data[:, :, 1] = 1
+        self.temperature.data[:, :, 3] = -1
+        self.temperature.data[:, :, 4] = -2
+        self.orography.data[:, :] = 10
+
+        result = LapseRate(nbhood_radius=1,
+                           min_lapse_rate=2 * DALR).process(
+            self.temperature,
+            self.orography,
+            self.land_sea_mask)
+
+        self.assertArrayAlmostEqual(result.data, expected_out)
+
+    def test_specified_max_and_min_lapse_rate(self):
+        """Test that the function correctly applies a specified, non default
+        maximum and minimum lapse rate."""
+        reset_cube_data(self.temperature, self.orography, self.land_sea_mask)
+
+        expected_out = np.array([[0.0392, 0.0392, 0.0, -0.0196, -0.0196],
+                                 [0.0392, 0.0392, 0.0, -0.0196, -0.0196],
+                                 [0.0392, 0.0392, 0.0, -0.0196, -0.0196],
+                                 [0.0392, 0.0392, 0.0, -0.0196, -0.0196],
+                                 [0.0392, 0.0392, 0.0, -0.0196, -0.0196]])
+
+        # West data points should be -3*DALR and East should be 2*DALR.
```

Comment:
Fixed the comment to read `-4*DALR`.

I have changed the wording in the LapseRate doc string to describe these min/max lapse rate values as _default_ lapse rate constraints. There is no reason why you shouldn't change these constraints (either higher or lower values), after all that's why they can be provided as optional arguments. So the it could be -`4*DALR` or `2*DALR` or whatever. The job of the plugin is then to constrain the lapse according to the input arguments (or `-3*DALR/DALR` as a _default_ if no arguments provided).

So I think its quite in order to test these values as there is nothing to stop them being specified as arguments which is part of the plugins functionality to allow different lapse rate constraints.